_data/simple-icons.json

```diff
@@ -11102,6 +11102,12 @@
             "hex": "E41D1B",
             "source": "https://www.reebok.com/us"
         },
+        {
+            "title": "Refine.dev",
+            "hex": "14141F",
+            "source": "https://www.figma.com/file/y8lhTyumAN0zYI5nKKLoLN/refine.dev-brand-assets?type=design&node-id=3588%3A22&mode=design&t=xoSXKRwKow3zJ2MC-1",
+            "guidelines": "https://www.figma.com/file/y8lhTyumAN0zYI5nKKLoLN/refine.dev-brand-assets?type=design&node-id=3588%3A22&mode=design&t=xoSXKRwKow3zJ2MC-1"
```

Comment:
File no longer exists. We could do with a permalink that the brand redirects to the relevant Figma file.

Reply:
IMO I think we're best to link to the homepage, as right-clicking on the logo will send you to the Figma file.

Reply:
@recepkutuk - do you have a permalink which redirects to the latest (like refine.dev/brand) that we can use?
It'd make reviewing in the future a _lot_ easier!

Reply:
Sure, here's the link: https://s.refine.dev/refine-brand-assets